iree/hal/cuda/dynamic_symbols.c

```diff
@@ -20,6 +20,14 @@ static const char* kCUDALoaderSearchNames[] = {
 #endif
 };
 
+static const char* kCUPTILoaderSearchNames[] = {
+#if defined(IREE_PLATFORM_WINDOWS)
+    // TODO (kooljblack): determine windows cupti dll
+#else
+    "libcupti.so"
+#endif
```

Comment:
style: add trailing `//  IREE_PLATFORM_WINDOWS` (similar to how namespace trailing }'s get the namespace name, preprocessor ifs do too; eg https://google.github.io/styleguide/cppguide.html#The__define_Guard